src/test/scala/CacheKeyTestSpec.scala

```diff
@@ -0,0 +1,114 @@
+import org.scalatest._
+
+import scala.sys.process._
+class CacheKeyTest {
```

Comment:
The functionality here should be pulled into the `Spec` class. Helper methods here should turn into helpers on the `Spec`.